packages/mdc-button/_mixins.scss

```diff
@@ -386,9 +387,12 @@ $query: mdc-feature-all()) {
 
   @include mdc-typography(button, $query);
   @include mdc-button-horizontal-padding($mdc-button-horizontal-padding, $query);
+  @include mdc-elevation-overlay-parent($query: $query);
+  @include mdc-elevation-overlay-size(100%, $query: $query);
 
   @include mdc-feature-targets($feat-structure) {
     display: inline-flex;
+    /* @alternate */
     position: relative;
```

Comment:
Can we remove `position: relative` from base if elevation mixin is already setting it?

Reply:
We could. My only concern is it's not obvious from the mixin that the positioning is `relative`. I'm open to suggestions though and happy to remove the default `position: relative` here.

Reply:
position relative is needed by both ripple & elevation. Do we need a separate mixin for parent position? We can document this instead.

Most of our components already have set to position relative.

WDYT?

Reply:
The separate mixin makes it clear which element will contain the elevation. Some elements, like the switch thumb or card, don't have position relative.

I'll remove the existing `position: relative;` from here and keep the mixin. Let me know if you think otherwise.